Issue with cmake

I receive this error while running `cd build && cmake ..`
```
cmake ..
-- CMAKE_CXX_COMPILER_ID: 
-- CMAKE_CXX_COMPILER_VERSION: 
-- CMAKE_SYSTEM: 
CMake Error at CMakeLists.txt:108 (if):
  if given arguments:

    "AND" "(" "NOT" ")" "AND" "(" "NOT" "FALSE" ")"
```

Changing 

`if (${PROFILE} AND NOT ${DEV} AND NOT ${MACOSX})` 

to 

`if ("${PROFILE}" AND NOT "${DEV}" AND NOT "${MACOSX}")` 

in CMakeLists.txt  fixed the issue for me.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -105,7 +105,7 @@ add_executable(centrolign_exec ${SRC_DIR}/main.cpp)
 target_link_libraries(centrolign_exec centrolign)
 set_property(TARGET centrolign_exec PROPERTY OUTPUT_NAME centrolign)
 
-if (${PROFILE} AND NOT ${DEV} AND NOT ${MACOSX})
+if ("${PROFILE}" AND NOT "${DEV}" AND NOT "${MACOSX}")
 	target_link_libraries(centrolign ${JEMALLOC_LIBRARIES})
 	target_link_libraries(centrolign_exec ${JEMALLOC_LIBRARIES})
 endif()

